conditions.php

```diff
@@ -4,102 +4,98 @@
-    "hours_since_created"               => "Hours since created",
-    "hours_since_updated"               => "Hours since updated",
-    "internal_ticket"                   => "Internal ticket",
-    "involved_operator"                 => "Involved operator",
-    "last_ticket_replier"               => "Last ticket replier",
-    "ticket_channel"                    => "Ticket channel",
-    "ticket_channel_account"            => "Ticket channel account",
-    "ticket_created"                    => "Ticket created",
-    "ticket_department"                 => "Ticket department",
-    "ticket_due_time"                   => "Ticket due time",
-    "ticket_message_content"            => "Ticket message content",
-    "ticket_message_count"              => "Ticket message count",
-    "ticket_priority"                   => "Ticket priority",
-    "ticket_sla_plan"                   => "Ticket SLA plan",
-    "ticket_status"                     => "Ticket status",
-    "ticket_subject"                    => "Ticket subject",
-    "ticket_tag"                        => "Ticket tag",
-    "ticket_updated"                    => "Ticket updated",
-    "user_email"                        => "User email",
-    "user_full_name"                    => "User full name",
-    "user_group"                        => "User group",
+    "assigned_operator"                 => "Dodeljeni agent",
+    "assigned_operator_group"           => "Dodeljena grupa agenata",
+    "hours_since_created"               => "Proteklo vreme od kreiranja tiketa",
+    "hours_since_updated"               => "Proteklo vreme od ažuriranja tiketa",
+    "internal_Tiket"                   => "Interni tiketi",
+    "involved_operator"                 => "Uključen agent",
+    "last_Tiket_replier"               => "Poslednji odgovor na tiket",
+    "Tiket_channel"                    => "Kanal tiketa",
+    "Tiket_channel_account"            => "Tiket nalog kanala",
+    "Tiket_created"                    => "Tiket kreiran",
+    "Tiket_department"                 => "Tiket odeljenje",
+    "Tiket_due_time"                   => "Vremenski rok tiketa",
+    "Tiket_message_content"            => "Sadržaj tiketa",
+    "Tiket_message_count"              => "Broj tiket poruka",
+    "Tiket_priority"                   => "Prioritet tiketa",
+    "Tiket_sla_plan"                   => "Tiket SLA plan",
+    "Tiket_status"                     => "Status tiketa",
+    "Tiket_subject"                    => "Naslov tiketa",
+    "Tiket_tag"                        => "Tiket tag",
+    "Tiket_updated"                    => "Tiket ažuriran",
+    "user_email"                        => "Korisnički email",
+    "user_full_name"                    => "Puno ime korisnika",
+    "user_group"                        => "Korisnička grupa",
 
     /*
      * 2.1.0
      */
-    "ticket_brand"                      => "Ticket brand",
+    "Tiket_brand"                      => "Tiket brand",
```

Comment:
Please could you fix lines like this which are causing the tests to fail. The correct version should be:
```php
"ticket_brand"                      => "Tiket brand",
```

Only the value on the right hand side between the quotes should be modified.